guides/doc-Provisioning_Virtual/master.adoc

```diff
@@ -0,0 +1,51 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: provisioning-virtual
+:provisioning-virtual:
+
+= {ProvisioningVirtualDocTitle}
+
+// This guide is not ready for stable releases
+ifdef::HideDocumentOnStable[]
+include::common/modules/snip_guide-not-ready.adoc[]
+endif::[]
+ifndef::HideDocumentOnStable[]
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+include::common/assembly_introduction-to-provisioning.adoc[leveloffset=+1]
+
+include::common/assembly_preparing-client-platforms.adoc[leveloffset=+1]
+
+include::common/assembly_preparing-templates-for-provisioning.adoc[leveloffset=+1]
+
+include::common/assembly_preparing-networking.adoc[leveloffset=+1]
+
+include::common/assembly_security-settings.adoc[leveloffset=+1]
+
+include::common/modules/con_provisioning-contexts.adoc[leveloffset=+1]
+
+include::common/modules/proc_setting-the-provisioning-context.adoc[leveloffset=+2]
+
+include::common/modules/proc_using-novnc-to-access-virtual-machines.adoc[leveloffset=+1]
+
+include::common/assembly_provisioning-virtual-machines-kvm.adoc[leveloffset=+1]
+
+include::common/assembly_provisioning-virtual-machines-rhv.adoc[leveloffset=+1]
+
+include::common/assembly_provisioning-virtual-machines-vmware.adoc[leveloffset=+1]
+
+ifndef::satellite[]
+include::common/assembly_provisioning-virtual-machines-proxmox.adoc[leveloffset=+1]
+endif::[]
+
+include::common/assembly_provisioning-virtual-machines-kubevirt.adoc[leveloffset=+1]
```

Comment:
I think we can remove this since the kubevirt plugin no longer works and will be removed in 6.17.